capstone/src/main/webapp/script.js

```diff
@@ -180,8 +180,7 @@ async function loadScheduledAnnouncements() {
   const color2 = '#BBB';
   var evenOdd = true;
   for (var announcement of json) {
-    const pictureSrc = await fetch('/get-image?blobKey=' + announcement.picture);
-    var pictureSrc;
+    var pictureSrc = await fetch('/get-image?blobKey=' + announcement.picture);
     if (announcement.picture) {
         pictureSrc = await fetch('/get-image?blobKey=' + announcement.picture);
```

Comment:
We don't need this line either, right? Because we already fetched pictureSrc, we just need pictureSrc.url or something

Reply:
Ah yes you're right, I can remove this as well. Thanks!